gravitee-node-license/src/main/java/io/gravitee/node/license/LicenseService.java

```diff
@@ -219,6 +219,11 @@ private String getLicenseFile() {
     }
 
     private void verify() throws Exception {
+        if (license == null) {
+            logger.debug("License will not be verified as it has not been loaded");
+            return;
+        }
+
```

Comment:
What could be the consequences when a valid license file is loaded and replaced with an invalid license file? Does the `license` attribute becomes `null` and could bypass the license check?

Reply:
Looks like the license is not hot reloaded right? So I assume if you update the file after the initial start, the license is not refreshed, isn't it ?

Reply:
It appears there is a file watcher running in the background. That's why I'm asking myself if the fix could have unexpected consequences.

Reply:
Ok, @ThibaudAV and I checked the different cases and added a missing improvement to not break the file watcher when the license file is corrupted on https://github.com/gravitee-io/gravitee-node/pull/214